appveyor/run_test.ps1

```diff
@@ -12,6 +12,11 @@ python -c "import platform; print(platform.architecture())"
 
 $wc = New-Object 'System.Net.WebClient'
 nosetests -s -v --with-ignore-docstrings --with-xunit --xunit-file=unit_results.xml .\tests\unit
+
+$env:MONKEY_PATCH_LOOP=1
```

Comment:
Is it possible to run a single command with an environment variable set? I'm concerned that I might come along in the future and add test commands below this without realizing they're running with `MONKEY_PATCH_LOOP` set.